lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -186,7 +186,7 @@ class @AutoEnrollmentViaCsv
     @$students_list_file = @$container.find("input[name='students_list']")
     @$csrf_token = @$container.find("input[name='csrfmiddlewaretoken']")
     @$results = @$container.find("div.results")
-    @$browse_button = @$container.find("#browseBtn-auto_enroll_csv")
+    @$browse_button = @$container.find("#browseBtn-auto-enroll")
```

Comment:
Note though that the revamp of the instructor dashboard is unlikely to go to this level of detail. It's more about converting the tab structure than doing a UI revamp of individual fields like this.

That being said, an ID is necessary to link a label with an input (as far as I know), unless the input is nested within the label. @clrux where you have introduced new IDs, can you instead nest the input within the label to avoid the introduction of an ID?